persistence/ravendb/outbox.md

```diff
@@ -31,6 +31,8 @@ snippet: OutboxRavendBTimeToKeep
 
 partial: disable-cleanup
 
+WARN: When running in [multi-tenant mode](/persistence/ravendb.md#multi-tenant-support) cleanup needs to be handled manually since NServiceBus can't know what databases are in use.
```

Comment:
```suggestion
WARN: When running in [multi-tenant mode](/persistence/ravendb.md#multi-tenant-support), cleanup needs to be manually handled since NServiceBus does not know what databases are in use.
```